service/src/db/dbOperations.ts

```diff
@@ -154,3 +154,31 @@ export async function batchInsert(artifacts: FhirArtifact[], action: string) {
   if (error) throw error;
   return results;
 }
+
+/**
+ * Updates a parent artifact and all of its children (if applicable) in a batch
+ * to set the date and approvalDate elements of the approved artifacts as well
+ * as add an 'Approve' cqfm-artifactComment extension to the artifacts
+ */
+export async function batchUpdate(artifacts: FhirArtifact[]) {
```

Comment:
I like that this is a `batchUpdate` that could be used for other update operations, but there's a bit of a mismatch in that it is currently pretty approval specific. Do we want to model this one more like `batchInsert`?